specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -252,26 +363,97 @@
         }
       ],
       "properties": {
+        "etag": {
+          "type": "string",
+          "description": "Etag of the alert rule."
+        },
         "properties": {
             "x-ms-client-flatten": true,
             "description": "Alert rule properties",
-            "$ref": "#/definitions/AlertRuleProperties"
+            "$ref": "#/definitions/ScheduledAlertRuleProperties"
         }
       }
     },
-    "AlertRuleProperties": {
+    "ScheduledAlertRuleProperties": {
       "type": "object",
       "description": "Alert rule property bag.",
       "properties": {
+        "ruleName": {
+          "type": "string",
+          "description": "The name for alerts created by this alert rule."
+        },
+        "description": {
+          "type": "string",
+          "description": "The description of the alert rule."
+        },
+        "severity": {
+          "type": "string",
+          "enum": ["Low", "Medium", "High", "Informational"],
+          "x-ms-enum": {
+            "name": "Severity",
+            "modelAsString": false
+          },
+          "description": "The severity for alerts created by this alert rule."
+        },
+        "enabled": {
+          "type": "boolean",
+          "description": "Determines whether this alert rule is enabled or disabled."
+        },
         "query": {
           "type": "string",
-          "description": "The query that will create alerts for this rule."
+          "description": "The query that creates alerts for this rule."
+        },
+        "queryFrequency": {
+          "type": "string",
+          "pattern": "^(\\d+\\.)?\\d{2}\\:\\d{2}\\:\\d{2}$",
+          "description": "TimeSpan format represents the frequncy for this alert rule to run."
         },
-        "period": {
+        "queryPeriod": {
```

Comment:
Microsoft.Insights named a similar concept 'windowSize'. Though you aren't supporting a constant floating window, you could call this 'queryWindowSize'. Consider aligning on terminology.